src/main/java/org/jboss/forge/arquillian/command/algeron/consumer/PactAlgeronConsumerTestSetup.java

```diff
@@ -33,6 +33,7 @@ public JavaClassSource updateTest(JavaClassSource test, String consumer, String
         annotateTestClass(test, consumer, provider);
         createContractFragment(test, fragmentName);
         createTestMethod(test, consumer, provider, fragmentName);
+        removeTestMethod("should_be_deployed", test);
```

Comment:
It doesn't make sense to me. Why do we have `should_be_deployed()` if there is no deployment method at all. I think this applies to `-standalone` mode and I believe we should make this template a bit more useful (or maybe just remove this empty method?)